### PR TITLE
update monitored ldiskfs to use lvm and ha

### PR DESCRIPTION
**Pull request by johnsonw**
update monitored ldiskfs to use lvm and ha

Signed-off-by: johnsonw <wjohnson@whamcloud.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2060)

### CODE REVIEW

File: iml-system-test-utils/src/lib.rs [hunk 1]
```diff
@@ -514,22 +514,36 @@ pub async fn configure_docker_network(config: &Config) -> Result<(), CmdError> {
...
+        }
+    });
 
-    try_join_all(xs).await?;
+    try_join_all(futures).await?;
+
+    let futures = config.storage_servers().into_iter().map(|x| {
+        tracing::debug!("setting up HA for {}", x);
+        async move {
+            vagrant::provision_node(x, "ha-ldiskfs-lvm-fs-setup")
```
Comment (jgrund) [inline iml-system-test-utils/src/lib.rs:537]: This is missing a step

Comment (johnsonw) [reply iml-system-test-utils/src/lib.rs:537]: Which step is this missing? The docs show:
```
 vagrant provision --provision-with=install-ldiskfs-no-iml,configure-lustre-network,create-ldiskfs-lvm-fs,ha-ldiskfs-lvm-fs-prep
 VBOX_PASSWD=<ROOT_PW_HERE>
 vagrant provision --provision-with=ha-ldiskfs-lvm-fs-setup
```
The `install-ldiskfs-no-iml` step is being done earlier and I was able to run `ha-ldiskfs-lvm-fs-setup` without the environment variable. Is there something else that's missing?

**Comment by jgrund** [comment]
> and I was able to run ha-ldiskfs-lvm-fs-setup without the environment variable

This breaks the VBox fence agent, which cannot communicate without the var being set

**Comment by johnsonw** [comment]
Ah I see. Ok I'll make sure to run the command with that environment variable set.

**Comment by jgrund** [comment]
> Ah I see. Ok I'll make sure to run the command with that environment variable set.

Make sure it's obfuscated and doesn't appear in the source or test output.

**Pull request closed without merging**